modules/client/examples/02-multisig-client/11-get-members.ts

```diff
@@ -16,12 +16,12 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Instantiate a Multisig plugin client.
 const multisigClient: MultisigClient = new MultisigClient(contextPlugin);
 
-const daoAddressorEns: string = "0x1234548357023847502348"; // or my-dao.dao.eth
+const daoPluginAddress: string = "0x1234548357023847502348"; // The address of the plugin that DAO has installed. You can find this through calling `getDao(daoAddress)` and getting the DAO details .
 
 const multisigMembers: string[] = await multisigClient.methods.getMembers(
-  daoAddressorEns,
+  daoPluginAddress,
 );
-console.log(multisigMembers);
+console.log({ multisigMembers });
```

Comment:
I agreed with @brickpop on leaving the `console.log` as a simple print withouut converting it wo an object in the places where only 1 thing is printed and there is no room for mistake